pubnub/models/subscription.py

```diff
@@ -18,11 +19,13 @@ class PNSubscribable:
     def __init__(self, pubnub_instance) -> None:
         self.pubnub = pubnub_instance
 
-    def subscription(self):
-        return Subscription(self.pubnub, self.subscription_names, self.subscription_type)
+    def subscription(self, *, timetoken: Optional[int] = None, region: Optional[str] = None,
```

Comment:
in other SDKs `.subscription()` doesn't take a timetoken or region

timetoken can be passed into `.subscribe()`

Reply:
This one is fixed